src/state/ingress.py

```diff
@@ -90,7 +93,10 @@ def from_provider(
                     )
                 backends.append(
                     HAProxyBackend(
-                        backend_name=backend_name, servers=servers, strip_prefix=strip_prefix
```

Comment:
As enabling server-side encryption should have its own PR, we can probably remove this.